Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Unital.lean

```diff
@@ -322,14 +322,16 @@ lemma cfc_cases (P : A → Prop) (a : A) (f : R → R) (h₀ : P 0)
     · rwa [cfc_apply_of_not_continuousOn _ h]
 
 variable (R) in
-lemma cfc_id : cfc (id : R → R) a = a :=
+lemma cfc_id (ha : p a := by cfc_tac) : cfc (id : R → R) a = a :=
```

Comment:
Maybe remove `ha` from Line 279, if it's never included automatically? Same for lines 280 and 281?

Reply:
These are fairly often included automatically, when they appear in the statements.